client-rest/src/app/transfer/initiate-transfer.component.ts

```diff
@@ -63,7 +63,9 @@ export class InitiateTransferComponent implements OnInit {
       let start: StartTransferJob = {
             id: this.progressService.transferId(),
             exportAuthData: this.progressService.exportAuthData(),
-            importAuthData: this.progressService.importAuthData()
+            importAuthData: this.progressService.importAuthData(),
+            // TODO: remove flag once client encrypts creds
+            authDataEncrypted: false
```

Comment:
Maybe debugAuthDataEncrypted?

Reply:
I'm probably missing something but could you explain why 'debug'? I was trying to make it mean "whether the auth data is encrypted when sent to the API servers", not necessarily a debug thing

Either way, I think we'll get rid of the flag soon since Jim is looking into encrypting creds in the demo client now